test/support/mocks/http_client_mock.ex

```diff
@@ -0,0 +1,51 @@
+defmodule WelliesWeb.HTTPClientMock do
```

Comment:
This would actually be a stub, not a mock, see:

1. https://martinfowler.com/articles/mocksArentStubs.html
2. https://stackoverflow.com/questions/3459287/whats-the-difference-between-a-mock-stub